include/swift/Basic/Features.def

```diff
@@ -514,6 +514,9 @@ EXPERIMENTAL_FEATURE(AllowRuntimeSymbolDeclarations, true)
 /// Allow use of `@cdecl`
 EXPERIMENTAL_FEATURE(CDecl, false)
 
+/// Allow use of `@extensible` on public enums
+SUPPRESSIBLE_EXPERIMENTAL_FEATURE(ExtensibleAttribute, false)
```

Comment:
This should just be a `SUPPRESSIBLE_LANGUAGE_FEATURE`, since you don't have to specify `-enable-experimental-feature ExtensibleAttribute` to write the attribute in source, right?